### PR TITLE
Added Meetings to DB

### PR DESCRIPTION
**Pull request by croz1007**
Added meetings to DB for use with Speakers later

### CODE REVIEW

File: lib/cbus_elixir/app/app.ex [hunk 1]
```diff
@@ -0,0 +1,104 @@
+defmodule CbusElixir.App do
+  @moduledoc """
+  The App context.
+  """
+
```
Comment (mdarby) [inline lib/cbus_elixir/app/app.ex:5]: This is the issue with re-generating a resource with the context; it clobbers any existing context with the same name. So if we use `App` (which I believe is correct) we need to make sure to append the generated changes with the existing code in `app.ex`

File: lib/cbus_elixir/app/meeting.ex [hunk 2]
```diff
@@ -0,0 +1,21 @@
+defmodule CbusElixir.App.Meeting do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias CbusElixir.App.Meeting
+
+
+  schema "meetings" do
+    field :address, :string
+    field :date, :string
+    field :location, :string
```
Comment (mdarby) [inline lib/cbus_elixir/app/meeting.ex:10]: Only need :date

File: lib/cbus_elixir/app/app.ex (hunk 1 above)
Comment (croz1007) [reply lib/cbus_elixir/app/app.ex:5]: I branched from master last night so none of your stuff was ever in there.

File: lib/cbus_elixir_web/router.ex [hunk 3]
```diff
@@ -19,8 +19,8 @@ defmodule CbusElixirWeb.Router do
     get "/", PageController, :index
   end
 
-  # Other scopes may use custom stacks.
-  # scope "/api", CbusElixirWeb do
-  #   pipe_through :api
-  # end
+  scope "/api", CbusElixirWeb do
+    pipe_through :api
+    resources "/meetings", MeetingController, except: [:new, :edit]
```
Comment (mdarby) [inline lib/cbus_elixir_web/router.ex:24]: I don't think we need routes; we'll just generate the meeting dates and we're done.

File: lib/cbus_elixir/app/app.ex (hunk 1 above)
Comment (mdarby) [reply lib/cbus_elixir/app/app.ex:5]: Sounds like a rebase off of master is needed now that it is in there?

File: lib/cbus_elixir_web/router.ex (hunk 3 above)
Comment (croz1007) [reply lib/cbus_elixir_web/router.ex:24]: That was all from the generator. I don't need any resources at all?

Comment (mdarby) [reply lib/cbus_elixir_web/router.ex:24]: Nope, no routes. We won't be managing Meetings at all outside of the initial generation

**Pull request closed without merging**